.github/workflows/deploy-happy-stack.yml

```diff
@@ -54,6 +54,10 @@ jobs:
           env: ${{ env.DEPLOYMENT_STAGE }}
           env-file: /home/runner/work/custom/envfile
           happy_version: "0.79.1"
```

Comment:
worth upgrading happy now as well? (is there a chance a later version of `happy` will block until the deployment is finished?)

Reply:
Not sure if that feature has ever been considered.